eras/conway/impl/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.20.0.0
 
+* Fix CDDL for `MultiAsset` in `TxOut` as well as the mint field.
```

Comment:
This needs to go into the `testlib` section, since it is huddle spec that has changed, rather than the actual decoders